src/hotspot/share/logging/logDiagnosticCommand.hpp

```diff
@@ -59,7 +59,6 @@ class LogDiagnosticCommand : public DCmdWithParser {
     return "Lists current log configuration, enables/disables/configures a log output, or rotates all logs.";
   }
 
-  // Used by SecurityManager. This DCMD requires ManagementPermission = control.
   static const JavaPermission permission() {
```

Comment:
Is any of this permission stuff still relevant? I couldn't figure out what ultimately looks at them. ??

Reply:
I don't know that.  It is passed by the MBean code.  It might be another (different) opportunity for a cleanup if the MBean code doesn't use it anymore.